core/css/tooltip.scss

```
@@ -28,6 +28,7 @@
     word-break: normal;
     word-spacing: normal;
     word-wrap: normal;
+	overflow-wrap: anywhere;
```

```suggestion
   overflow-wrap: anywhere;
```

tab vs space :see_no_evil: